scripts/upgrade

```diff
@@ -193,6 +194,12 @@ yunohost service add $app --description="Collaboration platform built for develo
 #=================================================
 ynh_script_progression --message="Starting a systemd service..." --weight=2
 
+# A lengthy database migration runs when upgrading from a version < 5.35.
+if dpkg --compare-versions "$previous_upstream_version" lt "5.35.0"
+then
+  ynh_print_warn --message="A database migration will now run. This may take a while..."
+fi
```

Comment:
@ericgaspar I added this to print a warning when upgrading (as the database migration may take between 1mn and 1h on moderately-sized instances).

Does the code looks OK to you?